readme.md

```diff
@@ -102,17 +102,37 @@ Start the spinner. Returns the instance.
 
 Stop and clear the spinner. Returns the instance.
 
-### .succeed()
+#### .succeed([text])
 
-Stop the spinner, change it to a green `✔` and persist the `text`. Returns the instance. See the GIF below.
+Stop the spinner, change it to a green `✔` and persist `text` (or `.text` if `text` is not provided). Returns the instance. See the GIF below.
 
-### .fail()
+#### .fail([text])
 
-Stop the spinner, change it to a red `✖` and persist the `text`. Returns the instance. See the GIF below.
+Stop the spinner, change it to a red `✖` and persist `text` (or `.text` if `text` is  not provided). Returns the instance. See the GIF below.
```

Comment:
> (or `.text` if `text` is  not provided)

Not clear enough what this means. Would be better to just explain it. For example:

> and persists the current text or `text` if provided.